src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Validators/WhenValidatingSelectProviderViewModel.cs

```diff
@@ -7,11 +8,28 @@ namespace SFA.DAS.EmployerCommitmentsV2.Web.UnitTests.Validators
 {
     [TestFixture]
     public class WhenValidatingSelectProviderViewModel
-    {
-        // check null
-        // check non numeric
-        // less than 1
-        // greater than 1
+    { 
+        [Test, MoqAutoData]
+        public void AndTheProviderIdIsNull_ThenReturnsInvalid(
+            SelectProviderViewModel viewModel,
+            SelectProviderViewModelValidator validator)
+        {
+            viewModel.ProviderId = null;
+            var result = validator.Validate(viewModel);
+
+            Assert.False(result.IsValid);
+        }
+
+        [Test, MoqAutoData]
+        public void AndTheProviderIdIsNonNumeric_ThenReturnsInvalid(
+            SelectProviderViewModel viewModel,
+            SelectProviderViewModelValidator validator)
+        {
+            viewModel.ProviderId = "abcdefghijklmnopqrstuvwxyz*!£$%^&*()_+¬`=-][#';/.,}{~@:?><";
+            var result = validator.Validate(viewModel);
+
+            Assert.False(result.IsValid);
+        }
 
         [Test, MoqAutoData]
```

Comment:
this test can now be removed or renamed, "0" isn't a default value.

Reply:
Removed test